CrowdStrike/crowdstrike-telemetry/tests/telemetry_event_31.json

```
@@ -0,0 +1,29 @@
+    "event": {
+      "type": ["info"],
+      "category": ["process"]
```

Map `event_type = ZeroTrustHostAssessment` to category host
```suggestion
      "category": ["host"]
```